server/config/config.go

```diff
@@ -777,6 +784,11 @@ func (c *ScheduleConfig) adjust(meta *configutil.ConfigMetaData, reloading bool)
 	if !meta.IsDefined("store-limit-mode") {
 		configutil.AdjustString(&c.StoreLimitMode, defaultStoreLimitMode)
 	}
+
+	if !meta.IsDefined("store-limit-version") {
```

Comment:
Do we need to consider reloading?

Reply:
In this pr, `""` is default config. It should be considered if the default config is setted as `v2`